generatorcore/bisko/__init__.py

```diff
@@ -18,24 +18,23 @@
 
 @dataclass
 class EnergySourceCalcIntermediate:
-
+    """
+    This class is a contains all relevant parameters for the conversion calculation from the "Einflussbilanz" to the "BISKO Bilanz" (See Readme) for a single energy source
+    like petrol or electricity. Einflussbilanz attributes (eb-prefix) are saved intermediatly for conveniance. 
+    """
+    #New attributes for Bisko:
+    energy: float = field(init=False)
```

Comment:
Why not just "energy: float", there should be no need to use field(init=False). But maybe I missed something?

Reply:
Somehow the typechecker complains if i call the constructur of "EnergySourceCalcIntermediate" only using 
```
energy: float
```
in the class definition instead of 

```
energy: float = field(init=False) 
```

![image](https://user-images.githubusercontent.com/98150321/170833624-47750ae5-cf92-44d9-8702-227cf9bfd4a9.png)